src/ts/components/hugeplaybacktogglebutton.ts

```diff
@@ -102,6 +103,14 @@ export class HugePlaybackToggleButton extends PlaybackToggleButton {
       firstPlay = false;
     });
 
+    player.addEventHandler(player.EVENT.ON_WARNING, (ev: WarningEvent) => {
```

Comment:
Please call the parameter `event`... there is no need to shorten the parameters and make them less readable because UglifyJS takes care of that in the production build